google/cloud/security/inventory/inventory_loader.py

```diff
@@ -189,6 +192,8 @@ def _build_pipelines(cycle_timestamp, configs, **kwargs):
             cycle_timestamp, configs, crm_api_client, project_dao),
         load_projects_buckets_pipeline.LoadProjectsBucketsPipeline(
             cycle_timestamp, configs, gcs_api_client, project_dao),
+        load_projects_buckets_acls_pipeline.LoadProjectsBucketsAclsPipeline(
```

Comment:
Actually, the order here matters, as some pipeline depends on other pipelines to run first: buckets, and then bucket_acls.